baremetal/metal3machine_manager.go

```diff
@@ -533,8 +534,35 @@ func (m *MachineManager) Delete(ctx context.Context) error {
 			host.Spec.NetworkData = nil
 			bmhUpdated = true
 		}
-		if host.Spec.Online {
+
+		//	Change bmh's online status to on/off  based on AutomatedCleaningMode and fastTrack values
+
+		//	AutomatedCleaningMode |	capm3FastTrack		|	BMH
+		//		disabled				false 			turn off
+		//		disabled				true 			turn off
+		//		metadata				false 			turn off
+		//		metadata				true 			turn on
+
+		//get capm3FastTrack from env
+		capm3FastTrack := os.Getenv("CAPM3_FAST_TRACK")
+		if capm3FastTrack == "" {
+			m.Log.Info("No CAPM3 FAST TRACK environment variable is given, setting it to false")
+			capm3FastTrack = "false"
+		}
```

Comment:
you should get the variable at setup time, since it won't change. So you could have this as a package variable and set before you add the reconciler in main.go

Reply:
If we declare camp3FastTrack inside main.go (main package), how we use it in baremetal package? 
Can please you clarify what do you mean by package variable?

Reply:
done